portfolio/src/main/webapp/script.js

```diff
@@ -166,4 +166,53 @@ function getImgName(imgsrc) {
     return splitname[0];
   }
   return "";
-}
\ No newline at end of file
+}
+
+// Create the script tag for map api and set the appropriate attributes
+var mapscript = document.createElement("script");
+mapscript.src = "https://maps.googleapis.com/maps/api/js?key=AIzaSyAy6bZSEG3FN2VdsI8vRcnyew6kt6BTsCg&callback=initMap";
+mapscript.defer = true;
+mapscript.async = true;
+
+// Attach callback function to display map to the `window` object
+window.initMap = function() {
+  // JS API is loaded and available
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: {lat: 39, lng: -39},
+    zoom: 3
+  });
```

Comment:
As well as zoom (users aren't sure why you chose 3). They would wonder why not 10, or 5, or 1, ect

Reply:
That's true. I added a comment line 179.